api/internal/core/store/validate.go

```diff
@@ -88,14 +89,17 @@ func NewAPISIXJsonSchemaValidator(jsonPath string) (Validator, error) {
 }
 
 func getPlugins(reqBody interface{}) map[string]interface{} {
-	switch reqBody.(type) {
+	switch bodyType := reqBody.(type) {
 	case *entity.Route:
+		log.Printf("type of reqBody: %#v", bodyType)
```

Comment:
use `log.infof` is better

Reply:
Sorry, the original log library does not support info, warn, etc.. levels, need to submit another PR to do this.